src/vsc_r_api.ts

```diff
@@ -116,6 +124,19 @@ export async function documentSave(id: string) {
   await targetDocument.save();
 }
 
+export async function documentSaveAll() {
+  await workspace.saveAll();
+}
+
+export function projectPath() {
+  const path = typeof workspace.workspaceFolders !== 'undefined' ? 
+    workspace.workspaceFolders[0] : undefined;
```

Comment:
We have support for multi-root workspace (i.e. working with multiple project folders).

A better approach here might be using `workspace.getWorkspaceFolder(document.uri)` where `document` is the last defined active document.